### PR TITLE
[Profiler] Fixed Name violation for Models class

### PR DESCRIPTION
**Pull request by Ghislain1**
## Description
Improve naming convention for properties in model classes

## Motivation and Context
For more Maintainbility  and Readability

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: tools/Profiler/NanoProfiler/.editorconfig.txt [hunk 1]
```diff
@@ -0,0 +1,276 @@
+# EditorConfig is awesome:http://EditorConfig.org
```
Comment (josesimoes) [inline tools/Profiler/NanoProfiler/.editorconfig.txt:1]: This shouldn't be in the PR, please remove and maybe add it to gitignore please.

**Review by josesimoes — CHANGES_REQUESTED** [review]
All good, just please remove that file and we're good to go.

File: tools/Profiler/NanoProfiler/.editorconfig.txt (hunk 1 above)
Comment (Ghislain1) [reply tools/Profiler/NanoProfiler/.editorconfig.txt:1]: This file allows to format code, normally we have to set it  for all projects.. Goal: each developer should use the same code formatter

Comment (josesimoes) [reply tools/Profiler/NanoProfiler/.editorconfig.txt:1]: I'm familiar with editorconfig file. 😉 

The project already has a code style defined. And stylecop rules set. If you're keen to have these setup, OK to ad stylecop on a separate PR.

Comment (Ghislain1) [reply tools/Profiler/NanoProfiler/.editorconfig.txt:1]: ok

Comment (josesimoes) [reply tools/Profiler/NanoProfiler/.editorconfig.txt]: @Ghislain1 this is been here for a while... just wanted to know if you'll be addressing the above comment. Thanks! :)

**Review by josesimoes — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**